drivers/flash/flash_stm32.h

```diff
@@ -35,8 +35,13 @@ bool flash_stm32_valid_range(off_t offset, u32_t len, bool write);
 int flash_stm32_write_range(unsigned int offset, const void *data,
 			    unsigned int len, struct flash_stm32_priv *p);
 
+#if defined(CONFIG_SOC_SERIES_STM32F4X)
+int flash_stm32_block_erase_loop(struct device *dev, unsigned int offset,
```

Comment:
remove the ifdef and just fixup the flash_stm32l4x.c version of flash_stm32_block_erase_loop to take the the same arguments.

Reply:
OK